store/store.go

```diff
@@ -70,6 +74,20 @@ func (s *storeImpl) Get(key string) (interface{}, bool) {
 	return object.contents, true
 }
 
+// GetByRegex returns objects from the store based on a regexp search.
+func (s *storeImpl) GetByRegex(expr string) (m map[string]interface{}) {
```

Comment:
What do you mean by "sounds a bit uncertain" ? Do you have something else in mind?